### PR TITLE
Bump Go, Helm and various dependencies

### PR DESCRIPTION
**Pull request by macedogm**
Bump Go, Helm and various dependencies to fix CVEs flagged by Trivy.

Signed-off-by: Guilherme Macedo <guilherme.macedo@suse.com>

### CODE REVIEW

**Comment by macedogm** [comment]
@mattfarina can you please take a look in this PR?

File: go.mod [hunk 1]
```diff
@@ -1,28 +1,28 @@
 module github.com/helm/helm-mapkubeapis
 
-go 1.18
+go 1.19
```
Comment (hickeyma) [inline go.mod:3]: Leave at 1.18 to sync with Helm project

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks @macedogm for pushing the updates. Small change and `mod tidy` and it should be good to go.

**Comment by hickeyma** [comment]
Sorry but closing this @macedogm as #84 had the changes done previous to you.

**Comment by macedogm** [comment]
@hickeyma no problem in closing this PR and thanks for merging the other one 🙏🏻

**Comment by macedogm** [comment]
@hickeyma forgot to ask, do you plan to cut a new release, please? If yes, when?

**Pull request closed without merging**